v0.11.0 blog post (#73)
endless-sky/endless-sky.github.io: _posts/2026-01-31-unstable-release-0.11.0.md

And now, here are some of the showcases of major changes for this update.
Wondering what all I meant by "quality of life" UI changes? Well, one of our new contributors, @xobes, has been hard at work looking at UI enhancements. For a long time, we've had requests to make some outfitter keybinds more easily accessible, especially when it comes to outfit storage. You will now find new buttons in the outfitter for installing/uninstalling outfits, loading/unloading them from cargo, and storing them planet-side! Not only this, but the Player Info panel is now easily accessible through a new button on the planetary UI panel, along with a few other UI adjustments.

<img class="centered shadowed" src="/images/blog/v0.11.0/shop-buttons.jfif" width="800" height="450" />

<img class="centered shadowed" src="/images/blog/v0.11.0/planet-panel.jfif" width="800" height="450" />

> Another long-standing request is to have more starts available to the game, which Anarchist has provided us! In addition to the default one currently in the game, there are now three more human starts for a little more variance - start richer, start with less, or start with something a bit different than what we have now. Which you choose is up to you!

Reviewer: ```suggestion
Another long-standing request is to have more starts available to the game, which @Anarchist2  has provided us! In addition to the default one currently in the game, there are now three more human starts for a little more variance - start richer, start with less, or start with something a bit different than what we have now. Which you choose is up to you!
```

<img class="centered shadowed" src="/images/blog/v0.11.0/starts.jfif" width="1058" height="636" />

> Something else Anarchist has done is heavily reworked bounty hunting jobs in human space. They are more common now, have lower combat rating requirements and a more gradual ease into more difficult jobs. The list of ships in their fleets has been overhauled, adding more variants and more usage of light warships in the lower end. Notably, bounty jobs now affect reputation, the same way hunting pirates does normally.

Reviewer: ```suggestion
Something else @Anarchist2 has done is heavily reworked bounty hunting jobs in human space. They are more common now, have lower combat rating requirements and a more gradual ease into more difficult jobs. The list of ships in their fleets has been overhauled, adding more variants and more usage of light warships in the lower end. Notably, bounty jobs now affect reputation, the same way hunting pirates does normally.
```

<img class="centered shadowed" src="/images/blog/v0.11.0/marauder.jfif" width="1092" height="671" />

> While we're still talking about human space, bene-dictator has implemented "message buoys" that send hails upon entering certain systems and traveling between regions, giving us a little more immersion when space truckin' around!

Reviewer: ```suggestion
While we're still talking about human space, @bene-dictator has implemented "message buoys" that send hails upon entering certain systems and traveling between regions, giving us a little more immersion when space truckin' around!
```

<img class="centered shadowed" src="/images/blog/v0.11.0/buoy.jfif" width="800" height="200" />

> Under the hood, several improvements to overall performance have been made and should noticably reduce gameplay lag. Pathfinding and casual flight should feel a lot smoother, and VRAM usage should be much lower using High DPI graphics. Thanks to the efforts of xobes, Amazininte, and warp-core!

Reviewer: ```suggestion
Under the hood, several improvements to overall performance have been made and should noticeably reduce gameplay lag. Pathfinding and casual flight should feel a lot smoother, and VRAM usage should be much lower using High DPI graphics. Thanks to the efforts of @xobes, @Amazinite , and @warp-core!
```

> Under the hood, several improvements to overall performance have been made and should noticeably reduce gameplay lag. Pathfinding and casual flight should feel a lot smoother, and VRAM usage should be much lower using High DPI graphics. Thanks to the efforts of @xobes, @Amazinite , and @warp-core!

Reviewer: ```suggestion
<img class="centered shadowed" src="/images/blog/v0.11.0/buoy.jfif" width="800" height="200" />

A new short, standalone story has been added to the Graveyard, investigating more about the Ka'het and their behaviors, thanks to @beccabunny! Look forward for more things to come in the Graveyard and Patir saga in the near future!

If you're a lore fanatic, there are now a few new hidden culture conversations around the galaxy. Should you come across one, you may gain a little insight into some of the galaxy's alien races and historical events. @Arachi-Lover 

Under the hood, several improvements to overall performance have been made and should noticeably reduce gameplay lag. Pathfinding and casual flight should feel a lot smoother, and VRAM usage should be much lower using High DPI graphics. Thanks to the efforts of @xobes, @Amazinite , and @warp-core!
```

---
layout: post
title: Unstable release 0.11.0
author: Saugia
---

Many of us are feeling that winter cold, but do you know what else is cold? The vacuum of space, where Endless Sky v0.11.0 awaits! All things new can be found in the [changelog](https://github.com/endless-sky/endless-sky/blob/v0.11.0/changelog) for this release.

> If you thought, "Hey, that's not v0.10.11!", you'd be right - we've decided to bump the version numbers again! Similar to how we updated version numbers for v0.10.0, we've achieved some notable records for Github development recently - the lowest number of open Pull Requests since 2020, and the lowest number of PRs older than one year since 2018! Additionally, there is a similar number of contributors and new PR's being opened (meaning that the lower PR numbers aren't a result of a proportional decrease in contributions), so overall, PRs are being resolved much quicker than in the past. On top of this, there are a few long-requested UI quality of life changes coming in v0.11.0 (more on that below!), so we felt this was a great time to increment the second version number again.

Reviewer: ```suggestion
If you thought, "Hey, that's not v0.10.17!", you'd be right - we've decided to bump the version numbers again! Similar to how we updated version numbers for v0.10.0, we've achieved some notable records for Github development recently - the lowest number of open Pull Requests since 2020, and the lowest number of PRs older than one year since 2018! Additionally, there is a similar number of contributors and new PR's being opened (meaning that the lower PR numbers aren't a result of a proportional decrease in contributions), so overall, PRs are being resolved much quicker than in the past. On top of this, there are a few long-requested UI quality of life changes coming in v0.11.0 (more on that below!), so we felt this was a great time to increment the second version number again.
```
